### PR TITLE
[ign-common3] Update BitBucket links

### PR DESCRIPTION
**Pull request by chapulina**
Signed-off-by: Louise Poubel <louise@openrobotics.org>

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -5,143 +5,143 @@
...
-    * [Pull request 185](https://bitbucket.org/ignitionrobotics/ign-common/pull-requests/185)
-    * [Issue 47](https://bitbucket.org/ignitionrobotics/ign-common/issue/47)
+    * [BitBucket pull request 185](https://osrf-migration.github.io/ignition-gh-pages/#!/ignitionrobotics/ign-common/pull-requests/185)
+    * [Issue 47](https://github.com/ignitionrobotics/ign-common/issues/47)
 
 1. Fix ColladaLoader to support mixamo models and fix skeleton animation loading
-    * [Pull request 179](https://bitbucket.org/ignitionrobotics/ign-common/pull-requests/179)
-    * [osrf/gazebo pull request 3071](https://bitbucket.org/osrf/gazebo/pull-requests/3071)
+    * [BitBucket pull request 179](https://osrf-migration.github.io/ignition-gh-pages/#!/ignitionrobotics/ign-common/pull-requests/179)
+    * [osrf/gazebo pull request 3071](https://osrf-migration.github.io/gazebo-gh-pages/#!/osrf/gazebo/pull-requests/3071)
```
Comment (mabelzhang) [inline Changelog.md:106]:
```suggestion
    * [BitBucket osrf/gazebo pull request 3071](https://osrf-migration.github.io/gazebo-gh-pages/#!/osrf/gazebo/pull-requests/3071)
```

File: Changelog.md [hunk 2]
```diff
@@ -5,143 +5,143 @@
...
 
 ## Ignition Common 3.2.0 (2019-08-07)
 
 1. Allow ignLogInit to use an absolute path. Added a timeToIso that converts a given time to an ISO string. Some console timestamps were using `IGN_SYSTEM_TIME_NS()` and others `ignition::common::systemTimeISO()`. Switched all to use `ignition::common::systemTimeISO()`.
-    * [Pull request 203](https://bitbucket.org/osrf/gazebo/pull-request/203)
+    * [BitBucket pull request 203](https://osrf-migration.github.io/gazebo-gh-pages/#!/osrf/gazebo/pull-requests/203)
 
 1. Port ColladaLoader fix: use default value of 1 for stride parameter when unset.
-    * [Pull request 196](https://bitbucket.org/osrf/gazebo/pull-request/196)
+    * [BitBucket pull request 196](https://osrf-migration.github.io/gazebo-gh-pages/#!/osrf/gazebo/pull-requests/196)
```
Comment (mabelzhang) [inline Changelog.md:78]: The original changelog linked to the wrong repo. These tickets are in ign-common, not gazebo.

```suggestion
    * [BitBucket pull request 196](https://osrf-migration.github.io/ignition-gh-pages/#!/ignitionrobotics/ign-common/pull-requests/196)
```

File: Changelog.md [hunk 3]
```diff
@@ -5,143 +5,143 @@
...
 
 1. Filesystem copy directory and parentpath
-    * [Pull request 200](https://bitbucket.org/osrf/gazebo/pull-request/200)
+    * [BitBucket pull request 200](https://osrf-migration.github.io/gazebo-gh-pages/#!/osrf/gazebo/pull-requests/200)
 
 ## Ignition Common 3.2.0 (2019-08-07)
 
 1. Allow ignLogInit to use an absolute path. Added a timeToIso that converts a given time to an ISO string. Some console timestamps were using `IGN_SYSTEM_TIME_NS()` and others `ignition::common::systemTimeISO()`. Switched all to use `ignition::common::systemTimeISO()`.
-    * [Pull request 203](https://bitbucket.org/osrf/gazebo/pull-request/203)
+    * [BitBucket pull request 203](https://osrf-migration.github.io/gazebo-gh-pages/#!/osrf/gazebo/pull-requests/203)
```
Comment (mabelzhang) [inline Changelog.md:75]:
```suggestion
    * [BitBucket pull request 203](https://osrf-migration.github.io/ignition-gh-pages/#!/ignitionrobotics/ign-common/pull-requests/203/page/1)
```

File: Changelog.md [hunk 4]
```diff
@@ -5,143 +5,143 @@
...
-    * [Pull request 197](https://bitbucket.org/osrf/gazebo/pull-request/197)
+    * [BitBucket pull request 197](https://osrf-migration.github.io/gazebo-gh-pages/#!/osrf/gazebo/pull-requests/197)
 
 1. ColladaLoader: fix wrong node weights caused by buffer overflow bug
-    * [Pull request 201](https://bitbucket.org/osrf/gazebo/pull-request/201)
+    * [BitBucket pull request 201](https://osrf-migration.github.io/gazebo-gh-pages/#!/osrf/gazebo/pull-requests/201)
 
 1. Filesystem copy directory and parentpath
-    * [Pull request 200](https://bitbucket.org/osrf/gazebo/pull-request/200)
+    * [BitBucket pull request 200](https://osrf-migration.github.io/gazebo-gh-pages/#!/osrf/gazebo/pull-requests/200)
```
Comment (mabelzhang) [inline Changelog.md:70]:
```suggestion
    * [BitBucket pull request 200](https://osrf-migration.github.io/ignition-gh-pages/#!/ignitionrobotics/ign-common/pull-requests/200)
```

File: Changelog.md [hunk 5]
```diff
@@ -5,143 +5,143 @@
...
-    * [Pull request 205](https://bitbucket.org/osrf/gazebo/pull-request/205)
+    * [BitBucket pull request 205](https://osrf-migration.github.io/gazebo-gh-pages/#!/osrf/gazebo/pull-requests/205)
 
 1. Add bvh animation support for skeleton and TrajectoryInfo
-    * [Pull request 197](https://bitbucket.org/osrf/gazebo/pull-request/197)
+    * [BitBucket pull request 197](https://osrf-migration.github.io/gazebo-gh-pages/#!/osrf/gazebo/pull-requests/197)
 
 1. ColladaLoader: fix wrong node weights caused by buffer overflow bug
-    * [Pull request 201](https://bitbucket.org/osrf/gazebo/pull-request/201)
+    * [BitBucket pull request 201](https://osrf-migration.github.io/gazebo-gh-pages/#!/osrf/gazebo/pull-requests/201)
```
Comment (mabelzhang) [inline Changelog.md:67]:
```suggestion
    * [BitBucket pull request 201](https://osrf-migration.github.io/ignition-gh-pages/#!/ignitionrobotics/ign-common/pull-requests/201)
```

File: Changelog.md [hunk 6]
```diff
@@ -5,143 +5,143 @@
...
 
 ## Ignition Common 3.3.0 (2019-08-27)
 
 1. skip GTSMeshUtils test on Windows due to issue #50
-    * [Pull request 205](https://bitbucket.org/osrf/gazebo/pull-request/205)
+    * [BitBucket pull request 205](https://osrf-migration.github.io/gazebo-gh-pages/#!/osrf/gazebo/pull-requests/205)
 
 1. Add bvh animation support for skeleton and TrajectoryInfo
-    * [Pull request 197](https://bitbucket.org/osrf/gazebo/pull-request/197)
+    * [BitBucket pull request 197](https://osrf-migration.github.io/gazebo-gh-pages/#!/osrf/gazebo/pull-requests/197)
```
Comment (mabelzhang) [inline Changelog.md:64]:
```suggestion
    * [BitBucket pull request 197](https://osrf-migration.github.io/ignition-gh-pages/#!/ignitionrobotics/ign-common/pull-requests/197)
```

File: Changelog.md [hunk 7]
```diff
@@ -5,143 +5,143 @@
...
 
 1. Support custom callbacks to find files on global interface
-    * [Pull request 226](https://bitbucket.org/ignitionrobotics/ign-common/pull-request/226)
+    * [BitBucket pull request 226](https://osrf-migration.github.io/ignition-gh-pages/#!/ignitionrobotics/ign-common/pull-requests/226)
 
 ## Ignition Common 3.3.0 (2019-08-27)
 
 1. skip GTSMeshUtils test on Windows due to issue #50
-    * [Pull request 205](https://bitbucket.org/osrf/gazebo/pull-request/205)
+    * [BitBucket pull request 205](https://osrf-migration.github.io/gazebo-gh-pages/#!/osrf/gazebo/pull-requests/205)
```
Comment (mabelzhang) [inline Changelog.md:61]:
```suggestion
    * [BitBucket pull request 205](https://osrf-migration.github.io/ignition-gh-pages/#!/ignitionrobotics/ign-common/pull-requests/205)
```

**Comment by chapulina** [comment]
> Fixed a few URLs

Thanks, @mabelzhang !

**Pull request merged**